Hugo docs build fails: 'then must be followed by a statement list' in github:download:solo:assets task

## Describe the bug

The Hugo documentation build fails during the `github:download:solo:assets` Taskfile task with a shell parse error:

```
Error: task: Failed to run task "build:all": task: Failed to run task "github:download:solo:assets": 30:118: `then` must be followed by a statement list
Error: Process completed with exit code 201.
```

**Root cause:** In [docs/site/Taskfile.github.yaml](https://github.com/hiero-ledger/solo/blob/main/docs/site/Taskfile.github.yaml), the `then` branch of the asset skip-condition contains only a commented-out `echo` line:

```sh
if [[ -f "$dest" || "$name" == example-* || ( "$name" == "docs-latest.tar.gz" && "$npmjsLatest" == "false" ) ]]; then
  # echo "✔ Skipping condition met for $name..."
else
  echo "⬇ Downloading $name..."
  curl ...
fi
```

Task uses the [`mvdan/sh`](https://github.com/mvdan/sh) Go shell interpreter rather than bash. Unlike bash, it does **not** treat a lone comment as a valid statement, so the `then` branch is effectively empty, producing `30:118: then must be followed by a statement list` (script line 30 = file line 67, where `then` sits at column 118).

## Describe the expected behavior

The Hugo docs build should parse and run the `github:download:solo:assets` task successfully and download release assets.

## To Reproduce

1. Trigger the Solo docs automation / Hugo doc build workflow.
2. Observe the build fail when running `github:download:solo:assets`.
3. See the error: `30:118: `then` must be followed by a statement list`.

Example failing run: https://github.com/hiero-ledger/solo/actions/runs/26824442849/job/79089415113

## Additional Context

Fix: invert the skip condition to use `continue` so the `then` branch is no longer empty, and run the download unconditionally afterward.

--- a/docs/site/Taskfile.github.yaml
+++ b/docs/site/Taskfile.github.yaml
@@ -62,17 +62,17 @@ tasks:
             url=$(echo "$asset" | jq -r '.url')
             dest="build/$tag/$name"
 
-            # skip if dest doesn't exist or if the artifact name starts with example- or if the filename is 
+            # skip if dest doesn't exist or if the artifact name starts with example- or if the filename is
             #  docs-latest.tar.gz and npmjsLatest is false
             if [[ -f "$dest" || "$name" == example-* || ( "$name" == "docs-latest.tar.gz" && "$npmjsLatest" == "false" ) ]]; then
               # echo "✔ Skipping condition met for $name, not writing $dest, npmjsLatest: $npmjsLatest"
-            else
-              echo "⬇ Downloading $name, $dest doesn't exist yet"
-              curl -sSL --retry 3 --fail -o "$dest" "$url" || {
-                echo "❌ Failed to download $url" >&2
-                rm -f "$dest"
-              }
+              continue
             fi
+            echo "⬇ Downloading $name, $dest doesn't exist yet"
+            curl -sSL --retry 3 --fail -o "$dest" "$url" || {
+              echo "❌ Failed to download $url" >&2
+              rm -f "$dest"
+            }
           done
         done
 

